packages/styles/components/_c.option-card.scss

```diff
@@ -0,0 +1,94 @@
+@use "sass:math";
+
+.mc-option-card {
+  $parent: get-parent-selector(&);
+
+  position: relative;
+  border-radius: get-border-radius("m");
+  cursor: pointer;
+
+  &__input {
+    position: absolute !important;
+    right: $mu100;
+    top: $mu100;
+    border-color: $color-input-border;
+
+    &.mc-radio__input,
+    &.mc-checkbox__input,
+    &:focus {
+      box-shadow: none;
+    }
+
+    &.mc-radio__input &.mc-checkbox__input &:checked {
+      border-color: $color-input-checked-border;
+    }
+
+    &.mc-radio__input &.mc-checkbox__input &:hover {
+      border-color: $color-input-border;
+    }
+
+    &.mc-radio__input &.mc-checkbox__input &:checked:hover {
+      border-color: $color-input-checked-border;
+    }
+
+    &:checked + #{$parent}__label {
+      border-color: $color-primary-01-500;
+      box-shadow: 0 0 0 $mu025 $color-success-100;
+    }
+
+    &:focus + #{$parent}__label {
+      &::after {
+        box-shadow: 0 0 0 2px $color-focus-border;
+      }
+    }
+
+    &:hover + #{$parent}__label {
+      box-shadow: 0 0 0 2px $color-grey-800;
+    }
```

Comment:
If you want this code to apply only to the hover state and not the checked state, it is best to put this code up first in your code.